experiment/process-e2e-suite-log/process-e2e-suite-log.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Initially wrote this helper script in bash (using unix tools -csplit, sed, tr- and bash basic file read/write) essentially because i knew how to do this exercise in bash quicker.
I can surely  do my best effort trying to write it down in golang if required for sharing. 
Thanks a lot

Reply:
We've found golang and python easier to extend, maintain and unit test. Will greatly appreciate if you can convert it!

Reply:
+1 to Go

Reply:
Thanks for asking me to write this in go. 
It has been an awesome exercise to work with, and a fruitful enlightening experience.
Please see the coments with the time measures. :)
Here's the [golang version](https://github.com/kubernetes/test-infra/pull/8640)
Thanks again!